oidc-wp/openid-connect-generic
# feat(#272): support ongoing update of user data from claims

### All Submissions:






### Changes proposed in this Pull Request:

Support ongoing update of user data from claims.
Currently, user data is only synced on first login, if the user does not yet exist in the wordpress database.
This PR triggers an update of first name, last name and email on each login.

Closes #272 .

### How to test the changes in this Pull Request:

1. Change user data of an existing wordpress user in your OAuth provider
2. Logout
3. Login
4. User data should have changed in Wordpress as well

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
Support ongoing update of user data from claims on each login
